docs/spec/spec.md

```diff
@@ -29,7 +29,7 @@ This specification elaborates on the `Persist CLI Tool` commands.
 ## 2. Initializing the Bal Project with Persistence Layer
 
 ```bash
-bal persist init
+bal persist init --datastore="datastore" --module="module_name"
```

Comment:
Could you update the spec with what the datastore and module params are meant to do?

Reply:
updated in [723d216](https://github.com/ballerina-platform/persist-tools/pull/148/commits/723d216a68224db64d79e239a736b48849e4706f)

Reply:
Shall we move it into table structure including following info?
1. Commnad
2. Description
3. Mandatory
4. Defaultvalue

Reply:
updated in [9da43ea](https://github.com/ballerina-platform/persist-tools/pull/148/commits/9da43ea9f1f785ffba65a086ba56ac2af1adefb3)